test/test_cache.py

```diff
@@ -458,7 +461,8 @@ def test_module_md_uniquify(self):
                 "arch": "noarch",
                 "profiles": ["default", "server"],
                 "installed_profiles": ["server"],
-                "status": "unknown"
+                "status": "unknown",
+                "active": True
```

Comment:
Could you please add unit test for the case, when there is one module active and another is inactive? Something like this:


```python
    def test_module_md_uniquify_active_and_inactive(self):
        modules_input = [
            {
                "name": "duck",
                "stream": 0,
                "version": "20180730233102",
                "context": "deadbeef",
                "arch": "noarch",
                "profiles": ["default"],
                "installed_profiles": [],
                "status": "enabled",
                "active": False
            },
            {
                "name": "duck",
                "stream": 0,
                "version": "20180730233102",
                "context": "deadbeef",
                "arch": "noarch",
                "profiles": ["default"],
                "installed_profiles": [],
                "status": "enabled",
                "active": True
            }
        ]

        expected_output = [
            {
                "name": "duck",
                "stream": 0,
                "version": "20180730233102",
                "context": "deadbeef",
                "arch": "noarch",
                "profiles": ["default"],
                "installed_profiles": [],
                "status": "enabled",
                "active": True
            }
        ]

        self.assertEqual(expected_output, ModulesProfile._uniquify(modules_input))
```